FV3/conf/configure.fv3.gnu_docker

```diff
@@ -17,7 +17,7 @@ DEBUG =
 REPRO = Y
 VERBOSE =
 OPENMP = Y
-AVX2 = N
+AVX2 = Y
```

Comment:
Are we sure that we are ok with switching this on? Validation?

Reply:
Are we always running on CPUs with AVX support?

Reply:
Initially we always ran with AVX support enabled.  Somewhere along the way it was switched off.  The checksum tests still pass with the AVX support enabled.

Reply:
Cool!